### DISCUSSION TITLE
Is there any way to run only one (or several) cpp unit tests?

### DISCUSSION THREAD

**Discussion by liorf95** [milvus-io/milvus#35290] [Q&A and General discussion] [answered]
Can I run only one (or several) cpp unit tests?

**Comment by yhmo** [comment, marked as answer]
Milvus cpp code is in this directory: /internal/core

Run this command:
`make build-cpp-with-unittest`

You will see a subdirectory generated: /internal/core/output/unittest with some binary files, such as "all_tests"

Run this binary with gtest filter.
For example, I want to run a single test case "NormalDistributionTest":
`./internal/core/output/unittest/all_tests --gtest_filter='SegmentCoreTest.NormalDistributionTest'`

**Reply by liorf95** [reply]
Great! Thanks.
